src/models/permissions.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const Permission = sequelize.define('Permissions', {
+    name: { type: DataTypes.STRING, primaryKey: true }
+  }, {
+    timestamps: false,
+    classMethods: {
+      associate: db => {
+        Permission.belongsToMany(db.Clients, { through: 'ClientPermissions' });
```

Comment:
Sorry if I was not clear. I totally agree with the `belongsToMany` relation.

What I mean is: I think it's enough to define it in Clients (https://github.com/mozilla-sensorweb/sensorweb-server/pull/71/files#diff-4727579a8668deee0b4103f2a0aa458bR48) and that it's (maybe) not useful to define it here _as well_.

The relation is already effective with the first command (I tried it myself -- I correctly get the relation table, and tests are passing). The second command only adds methods like `addClients` to a `Permission` object, or maybe find all clients that has a permission. I don't know if that's useful, but you know better.

That's all I'm saying here: asking if the bidirectional relation is useful :) Again, I agree with you thinking that `belongsToMany` is the right relation type.